### ISSUE TITLE
Brand new windows 10 install, TF doesn't run: ModuleNotFoundError: No module named '_pywrap_tensorflow_internal'

### ISSUE THREAD

**Issue by falconair** [tensorflow/tensorflow#13203]
I'm submitting this as a bug, instead of a support question on SO, because I'm getting this error after a very clean install of the operating system. The error occurs when I attempt to do the most basic validation, but just importing tensorflow!

------------------------

### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
No custom code, the error is occurring when I do `import tensorflow as tf` at the repl 

- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
Windows 10 Home, version 1703, fresh install (the only other thing installed on the computer is Chrome and Anaconda distribution of Python 3.6)

- **TensorFlow installed from (source or binary)**:
Binary. Followed the instruction to the letter, except type in 'pip' instead of 'pip3', as in:
`pip3 install --upgrade tensorflow-gpu` (this is after installing cuda and cudablas)

- **TensorFlow version (use command below)**:
1.3 (whatever is in the repo as of about 30 minutes ago)

- **Python version**:  Python 3.6.1 :: Anaconda custom (64-bit)

- **Bazel version (if compiling from source)**:
- **CUDA/cuDNN version**:
>nvcc --version
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2016 NVIDIA Corporation
Built on Mon_Jan__9_17:32:33_CST_2017
Cuda compilation tools, release 8.0, V8.0.60

- **GPU model and memory**:
NVIDIA GeForce GTX 960M
Approx. Total Memory: 18313 MB

- **Exact command to reproduce**:
import tensorflow as tf

### Describe the problem
I re-imaged my windows 10 laptop, installed Chrome, installed the latest anaconda python 3.6 and attempted to install tensorflow following the official instructions on the web page (all of this within the last hour or two of filing this ticket).

Installed Cuda and Cudablas from nvidia's website, as required by TF instructions.
Installed tensorflow-gpu

Saw an exception related to some setuptools file not being found (sorry, didn't keep a record of that)
Upgraded setuptools

Did a reinstall of tensorflow, but added `--force-reinstall` flag, which claimed to have installed tensorflow without errors:

Successfully installed bleach-1.5.0 html5lib-0.9999999 markdown-2.6.9 numpy-1.13.1 protobuf-3.4.0 setuptools-36.5.0 six-1.11.0 tensorflow-gpu-1.3.0 tensorflow-tensorboard-0.1.6 werkzeug-0.12.2 wheel-0.30.0

Ran the import command and saw these exception trace:

### Source code / logs
```
(C:\Users\someuser\Anaconda3) C:\Users\someuser>python
Python 3.6.1 |Anaconda custom (64-bit)| (default, May 11 2017, 13:25:24) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import tensorflow as tf
Traceback (most recent call last):
  File "C:\Users\someuser\Anaconda3\lib\site-packages\tensorflow\python\pywrap_tensorflow_internal.py", line 18, in swig_import_helper
    return importlib.import_module(mname)
  File "C:\Users\someuser\Anaconda3\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 978, in _gcd_import
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load
  File "<frozen importlib._bootstrap>", line 950, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 648, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 560, in module_from_spec
  File "<frozen importlib._bootstrap_external>", line 922, in create_module
  File "<frozen importlib._bootstrap>", line 205, in _call_with_frames_removed
ImportError: DLL load failed: The specified module could not be found.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\someuser\Anaconda3\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 41, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "C:\Users\someuser\Anaconda3\lib\site-packages\tensorflow\python\pywrap_tensorflow_internal.py", line 21, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "C:\Users\someuser\Anaconda3\lib\site-packages\tensorflow\python\pywrap_tensorflow_internal.py", line 20, in swig_import_helper
    return importlib.import_module('_pywrap_tensorflow_internal')
  File "C:\Users\someuser\Anaconda3\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ModuleNotFoundError: No module named '_pywrap_tensorflow_internal'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\someuser\Anaconda3\lib\site-packages\tensorflow\__init__.py", line 24, in <module>
    from tensorflow.python import *
  File "C:\Users\someuser\Anaconda3\lib\site-packages\tensorflow\python\__init__.py", line 49, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "C:\Users\someuser\Anaconda3\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 52, in <module>
    raise ImportError(msg)
ImportError: Traceback (most recent call last):
  File "C:\Users\someuser\Anaconda3\lib\site-packages\tensorflow\python\pywrap_tensorflow_internal.py", line 18, in swig_import_helper
    return importlib.import_module(mname)
  File "C:\Users\someuser\Anaconda3\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 978, in _gcd_import
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load
  File "<frozen importlib._bootstrap>", line 950, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 648, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 560, in module_from_spec
  File "<frozen importlib._bootstrap_external>", line 922, in create_module
  File "<frozen importlib._bootstrap>", line 205, in _call_with_frames_removed
ImportError: DLL load failed: The specified module could not be found.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\someuser\Anaconda3\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 41, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "C:\Users\someuser\Anaconda3\lib\site-packages\tensorflow\python\pywrap_tensorflow_internal.py", line 21, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "C:\Users\someuser\Anaconda3\lib\site-packages\tensorflow\python\pywrap_tensorflow_internal.py", line 20, in swig_import_helper
    return importlib.import_module('_pywrap_tensorflow_internal')
  File "C:\Users\someuser\Anaconda3\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ModuleNotFoundError: No module named '_pywrap_tensorflow_internal'


Failed to load the native TensorFlow runtime.

See https://www.tensorflow.org/install/install_sources#common_installation_problems

for some common reasons and solutions.  Include the entire stack trace
above this error message when asking for help.
```

**Comment by shivaniag** [comment]
This is a duplicate of #13196. Please see that issue for more information.

**Comment by falconair** [comment]
Hi @shivaniag , thanks for your quick response. 

While I'm still trying to get this issue resolved, I created a ticket on github because following exact instructions from TF's website on a brand new machine, pretty standard,  shouldn't cause exception traces. The fact that I wasn't able to run it means there is a 'bug' some where in the docs or the code.

Regarding the error, it doesn't look like this is the exact issue as #13196. There someone was apparently installing from source, which is why git shows up in their stack trace. I don't think that happens in my case (a quick ctrl+f, at least, didn't show it).

This issue is actually all over tensorflow's bug tracker under different names (now that I look more closely). For example, here is another one #8385, where the suggested solution is to install VC++ redistributable (which I also did, and which did not resolve the issue).
